README.md

```diff
@@ -12,209 +12,9 @@
-
-```console
-gtctl cluster create mycluster --bare-metal
-```
-
-It will create all the meta information on `${HOME}/.gtctl`.
-
-If you want to do more configurations, you can use the yaml format config file:
-
-```console
-gtctl cluster create mycluster --bare-metal --config <your-config-file>
-```
-
-You can refer to the example [`cluster.yaml`](./examples/bare-metal/cluster.yaml) and [`cluster-with-local-artifacts.yaml`](./examples/bare-metal/cluster-with-local-artifacts.yaml).
-
-### Deploy in Kubernetes
-
-#### Prerequisites
-
-- **Kubernetes 1.18 or higher version is required.**
-
-  You can use the [`kind`](https://kind.sigs.k8s.io/) to create your own Kubernetes cluster:
-
-  ```console
-  kind create cluster
-  ```
-
-#### Cluster Operations
-
-Create your own GreptimeDB cluster and etcd cluster:
-
-```console
-gtctl cluster create mycluster -n default
-```
-
-If you want to use artifacts(charts and images) that are stored in the CN region, you can enable `--use-greptime-cn-artifacts`:
-
-```console
-gtctl cluster create mycluster -n default --use-greptime-cn-artifacts
-```
-
-After creating, the whole GreptimeDB cluster will start in the `default` namespace:
-
-```console
-# Get the cluster.
-gtctl cluster get mycluster -n default
-
-# List all clusters.
-gtctl cluster list
-```
-
-You can use the `kubectl port-forward` command to forward frontend requests:
-
-```console
-kubectl port-forward svc/mycluster-frontend 4002:4002 > connections.out &
-```
-
-Use your `mysql` client to connect to your cluster:
-
-```console
-mysql -h 127.0.0.1 -P 4002
-```
-
-If you want to delete the cluster, you can:
-
-```console
-# Delete the cluster.
-gtctl cluster delete mycluster -n default
-
-# Delete the cluster, including etcd cluster.
-gtctl cluster delete mycluster -n default --tear-down-etcd
-```
-
-#### Dry Run Mode
-
-`gtctl` provides `--dry-run` option in cluster creation. If a user executes the command with `--dry-run`, `gtctl` will output the manifests content without applying them:
-
-```console
-gtctl cluster create mycluster -n default --dry-run
-```
-
-#### Experimental Feature
-
-You can use the following commands to scale (or down-scale) your cluster:
-
-```console
-# Scale datanode to 3 replicas.
-gtctl cluster scale <your-cluster> -n <your-cluster-namespace> -c datanode --replicas 3
-
-# Scale frontend to 5 replicas.
-gtctl cluster scale <your-cluster> -n <your-cluster-namespace> -c frontend --replicas 5
-```
-
-## Development
-
-There are many useful tools provided through Makefile, you can simply run `make help` to get more information:
-
-- Compile the project
-
-  ```console
-  make
-  ```
-
-  Then the `gtctl` will be generated in `./bin/`.
-
-
-- Run the unit test
-
-  ```console
-  make test
-  ```
-
-- Run the e2e test
-
-  ```console
-  make e2e
-  ```
+* [Quickstart](https://docs.greptime.com/user-guide/operations/gtctl) to install and use gtctl
```

Comment:
I think keeping it in readme is better since it can let users know the basic operations at first glance.

We can keep the basic and general operations in readme and leave more details on our docs website.

Reply:
guess we can keep `one-line installation` and `playground` section, leave other in the docs website.

Reply:
sounds reasonable.